openreview/venue_request/process/revisionProcess.py

```diff
@@ -17,9 +17,9 @@ def process(client, note, invitation):
         conference.setup_post_submission_stage()
 
     if invitation_type == 'Bid_Stage':
-        conference.setup_matching(build_conflicts=True)
+        conference.setup_matching(build_conflicts=('Conflict Detection' in forum_note.content.get('Paper Matching', [])))
```

Comment:
what happens with the current venues that don't have `Conflict Detection` option?

Also I think the manual assignment in the PC console is checking the conflicts, right? so may be the conflict computation should be always true.

Reply:
Ok, I have removed the option for Conflict Detection from the request form and we are now always building conflicts.